src/zjs_frdm_k64f.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
zjs_k64f_pins.json

Do `ls` and look over the list to see if there are other one-offs.